masonite-orm/models.md

```diff
@@ -294,6 +294,41 @@ You can disable this behavior as well:
 User.with_trashed().all() #== SELECT * FROM `users`
 ```
 
+### Changing Primary Key to use UUID
+
+Masonite ORM also comes with another global scope to enable using UUID as primary keys for your models.
+
+Simple inherit the `UUIDPrimaryKey` scope:
+
+```python
+from masoniteorm.scopes import UUIDPrimaryKeyMixin
+
+class User(Model, UUIDPrimaryKeyMixin):
+  # ..
+```
+
+You should also define a UUID column with primary constraint in a migration
```

Comment:
You say primary constraint here but I don't see the primary constraint?

Reply:
I had a question about this, cool that you're pointing it out. Is a primary key not automatically assigned to the field named "id" ?
Also can we currently do this ?  `table.uuid("id").primary()` ? or should we do : 
```
table.uuid("id")
table.primary("id")
```

Reply:
No we can't assume that id is the primary column. I typically don't use id as primary columns actually.  For work we typically do user_id or order_id as the primary auto incrementing columns

Reply:
We don't currently support the first way of setting a primary key but we would be able to add it in like 30 seconds. We currently do that with other fields like unique constraints

Reply:
Would be cleaner to be able to add it this way

Reply:
I agree. Can open a ticket. Would just have to copy the way the unique method works in the blueprint class

Reply:
I'll try to do it in a small PR and then merge the doc PR

Reply:
Done here https://github.com/MasoniteFramework/orm/pull/208